pkg/yurtctl/cmd/yurtinit/phases/install_flannel.go

```diff
@@ -0,0 +1,65 @@
+/*
+Copyright 2021 The OpenYurt Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package phases
+
+import (
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/kubernetes/cmd/kubeadm/app/cmd/phases/workflow"
+	kubeContants "k8s.io/kubernetes/cmd/kubeadm/app/constants"
+
+	"github.com/openyurtio/openyurt/pkg/yurtctl/constants"
+	kubeutil "github.com/openyurtio/openyurt/pkg/yurtctl/util/kubernetes"
+)
+
+func NewInstallFlannelPhase() workflow.Phase {
+	return workflow.Phase{
+		Name:  "Install flannel.",
+		Short: "Install flannel.",
+		Run:   runInstallFlannel,
+	}
+}
+
+func runInstallFlannel(c workflow.RunData) error {
```

Comment:
Why is the version of flannel hardcoded?

Reply:
fixed